git ignore patch

Ignore some files generated at build time.

diff --git a/.gitignore b/.gitignore
index a163728..419441f 100644
--- a/.gitignore
+++ b/.gitignore
@@ -78,3 +78,5 @@ cscope.*
 
 # spl ais files
 /spl/*.ais
+
+include/configs/onie_version.h
